manifests/charts/istiod-remote/templates/mutatingwebhook.yaml

```diff
@@ -15,8 +15,8 @@ metadata:
 webhooks:
   - name: sidecar-injector.istio.io
     clientConfig:
-      {{- if .Values.sidecarInjectorWebhook.injectionURL }}
-      url: {{ .Values.sidecarInjectorWebhook.injectionURL }}
+      {{- if .Values.injectionURL }}
+      url: {{ .Values.injectionURL }}
```

Comment:
I thought this would be `.Values.istiodRemote.injectionURL`? But I would think it fails the tests if it should be so I am a bit confused..

Reply:
because it is in the same chart, same as how we refer to revision, e.g. line 12 `   istio.io/rev: {{ .Values.revision | default "default" }}`

Reply:
tested too `helm template manifests/charts/istiod-remote manifests/charts/istiod-remote -f manifests/charts/global.yaml --set injectionURL="https://linsun:15017/inject"` and validated the injectionURL passed in correctly

Reply:
@howardjohn your comment was correct.  While this reads fine for helm template but didn't read well for istiod remote component.  https://github.com/istio/istio/pull/23793/files